catalog/src/test/java/org/killbill/billing/catalog/dao/TestCatalogOverrideDao.java

```diff
@@ -94,5 +102,56 @@ public void testGetOverriddenPlanPhases() throws Exception {
         }
     }
 
+    @Test(groups = "slow")
+    public void testGetOverriddenPlanPhasesWithUsageOverrides() throws Exception {
+
+        final StandaloneCatalog catalog = XMLLoader.getObjectFromString(Resources.getResource("SpyCarAdvanced.xml").toExternalForm(), StandaloneCatalog.class);
+        final Plan plan = catalog.findCurrentPlan("gas-monthly");
+
+        final PlanPhasePriceOverride[] resolvedOverrides = new PlanPhasePriceOverride[plan.getAllPhases().length];
+
+        List<TieredBlockPriceOverride> tieredBlockPriceOverrides = new ArrayList<TieredBlockPriceOverride>();
+        DefaultTieredBlockPriceOverride tieredBlockPriceOverride = new DefaultTieredBlockPriceOverride("gallons",new Double("1"), new BigDecimal(4), new Double("100"));
```

Comment:
Hey @pierre : Yes that makes sense to me. But I have added currency to the api instead of changing price to DefaultInternationalPrice. I updated the pull request with the same.Please let me know if that would work?